site/content/api/Command-And-Control-API.md

```diff
@@ -40,9 +40,18 @@ the *Business Application* needs to open a receiver link as precondition:
 
 1. The *Business Application* has established an AMQP link in role *receiver* with the source address `control/${tenant_id}/${reply-id}`.
 This link is used by the *Business Application* to receive the response of the executed command. This link’s source address is also referred to as `the reply-to address` for the request messages. 
-2. The `${reply-id}` may be any arbitrary string chosen by the client.  
+2. The `${reply-id}` may be any arbitrary string chosen by the client. 
+
+### Choosing a strategy for building the receiver link address
+
+While the `${reply-id}` can be completely chosen by the client, the specific use case for sending commands may influence 
+how it should be set.
+Thus following hints for choosing the right `${reply-id}` may be helpful:
+
+- if only one command response is expected from the device, the `${reply-id}` could be set to a UUID and closed again after receiving the response.
```

Comment:
Since the reply-id is something that need to be given to the device and will be send back I do not think that the advice for a UUID is the best. The replyTo is already scoped to tenant and device, so in most cases a UUID is much more then needed. Your phrase "arbitrary string to scope the command sender instance" would also be true.